### PR TITLE
Task Adding Tasks

### PR DESCRIPTION
**Pull request by nguyenngon95**
Task Adding Tasks

### CODE REVIEW

**Comment by nguyenngon95** [comment]
Ready

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -15,4 +16,19 @@ public function index(Request $request)
...
     }
+
+    public function store(TaskRequest $request)
+    {
+        $result = $request->user()->tasks()->create([
+            'name' => $request->name,
+        ]);
+
+        if ($result) {
+            $request->session()->flash('suc', 'Success');
```
Comment (tranducquoc) [inline app/Http/Controllers/TaskController.php]: i18n

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -15,4 +16,19 @@ public function index(Request $request)
...
+        } else {
+            $request->session()->flash('err', 'Fail');
```
Comment (tranducquoc) [inline app/Http/Controllers/TaskController.php]: tương tự

**Comment by tranducquoc** [comment]
commented

**Comment by nguyenngon95** [comment]
Ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**